static/src/js/components/ErrorBanner/ErrorBanner.jsx

```diff
@@ -32,7 +22,6 @@ export const ErrorBannerContainer = ({
 
   const { disableDatabaseComponents } = getApplicationConfig()
 
-  // Suppress error if it is a result of the database being under maintenance
```

Comment:
I mean the comment is still "true" in the long run we maybe wanna get rid of that code it was for the very specific purpose of when we had to disable the database in prod to migrate to the encrypted version

Reply:
I'll re-add it